rules/ke_rules.yaml

```diff
@@ -61,13 +61,25 @@ rules:
     check_serverconfig_valid_master_versions: false
     allowed_nodepool_versions:
       - major: '1.8'
-        minor: '12-gke.1'
+        minor: '10-gke.2'
+        operator: '='
```

Comment:
Can we do a `>=` operator here, so that it will capture `1.8.12-gke.3` and not repeat the major versions?

Reply:
Same with the ones below too.

Reply:
After clarifying this offline, can we just a comment here to explain the gap, so that it's easier to understand?